### ISSUE TITLE
Add host to socket.io

### ISSUE THREAD

**Issue by martingg88** [balderdashy/sails.io.js#24]
Is there any option to add host to socket.io.js before it request to /socket.io?

With this option that allow me to put this socket.io.js in cross domain platform. 
besides that i can't prevent from following issue. 
GET /socket.io/1/?__sails_io_sdk_version=0.10.0&__sails_io_sdk_platform=browser&__sails_io_sdk_language=javascript&t=1409979425989 404 (Not Found)

**Comment by safani** [comment]
Bump

**Comment by daedalus28** [comment]
The io.sails.url method is correct according to the [documentation](https://github.com/balderdashy/sails.io.js#cross-domain). If it's not working, it's possible that the autoconnect is happening before you have a chance to set it. If you're in a RequireJS environment, this will be fixed with #61

**Comment by sailsbot** [comment]
Thanks for posting, @martingg88.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
